plugin/backend_linux.go

```diff
@@ -259,10 +265,35 @@ func (pm *Manager) Pull(ctx context.Context, ref reference.Named, name string, m
 }
 
 // List displays the list of plugins and associated metadata.
-func (pm *Manager) List() ([]types.Plugin, error) {
+func (pm *Manager) List(pluginFilters filters.Args) ([]types.Plugin, error) {
+	if err := pluginFilters.Validate(acceptedPluginFilterTags); err != nil {
+		return nil, err
+	}
+
+	enabledOnly := false
```

Comment:
I think you misunderstood my comment :sweat_smile: I just think we don't support `1` and `0` for boolean filters (`--filter dangling=true` works, `--filter dangling=1` does not).